.github/workflows/docker_update_theia.yml

```diff
@@ -0,0 +1,52 @@
+# Anyone with write permissions to the antrea-io/theia Github repository can
```

Comment:
didn't you add this workflow also in the commit for #3 ?

Reply:
Yes, removed it here.